src/AiidaDFTK.jl

```diff
@@ -128,9 +129,13 @@ function run_json(filename::AbstractString; extra_output_files=String[])
     # Threading setup ... maybe later need to take parameters
     # from the JSON into account
     if mpi_nprocs() > 1
-        disable_threading()
-    else
-        setup_threading()
+        # Julia threads are controlled when launching Julia and should be disabled by the user.
+        if Threads.nthreads() > 1
+            @warn("Running through MPI, but threading was not disabled!"
```

Comment:
Actually if DFTK threads is 1 than this is also ok here (because than DFTK simply does not use the threads ... which are in principle available for other things). Maybe just check that ?

Reply:
done